src/webview.js

```
@@ -55,7 +55,7 @@ export default class WebView extends React.Component {
   }
 
   render() {
-    const tagProps = filterKeys(this.props, propName => propName in tagPropTypes)
+    const tagProps = filterKeys(this.props, propName => !(propName in eventPropTypes))
```

Ah, interesting. It's probably better this way so we can still pass in `className`, `style`, etc.

---

yeah. if there’s an official set of common HTML attributes and their proptypes we can access, that would probably be better, but until now, silently attaching arbitrary attributes to `webview` will be better than silently discarding them